### ISSUE TITLE
Domain Error 

### ISSUE THREAD

**Issue by Rezaariaa** [notthebee/ansible-easy-vpn#292]
Hello sir, 
i used this book on my server but i cant config, when i input my domain , i have error, this error said ip not configured on my domain . but when i ping my domain i can see ip correctly , where is problem sir?

**Comment by notthebee** [comment]
The subdomains adguard.DOMAIN, wg.DOMAIN and auth.DOMAIN should also point to the VPS. Is that the case for you?

**Comment by iPujhin** [comment]
I have this problem, can you help me? I created a service in AWS and released ports 80,443,53,51820,22. my free domain is from duckdns, I already indicated my IP but even so, during installation, the following error message appears: DNS problem: SERVFAIL looking for CAA for deviceconnectionvpn.duckdns.org - the domain nameservers may be faulty

**Comment by notthebee** [comment]
@sekhar989 DuckDNS automatically creates a wildcard CNAME record for the subdomains. 
So when you create an example.duckdns.org domain, *.example.duckdns.org will automatically resolve to the same IP address

**Comment by sekhar989** [comment]
> @sekhar989 DuckDNS automatically creates a wildcard CNAME record for the subdomains. So when you create an example.duckdns.org domain, *.example.duckdns.org will automatically resolve to the same IP address

Ok ok! However, I'm still facing the issue. Please take a look at the screenshots for details.
<img width="1240" alt="Screenshot 2024-07-04 at 14 02 32" src="https://github.com/notthebee/ansible-easy-vpn/assets/19548050/0be89208-3f7a-4712-ae9b-1cbe036beec6">
![image](https://github.com/notthebee/ansible-easy-vpn/assets/19548050/2e0f2556-4ce9-45fd-9ee5-4907579008f7)

Firstly, I appreciate the hard work and sharing it with us. Thank You so much for this.
I'm a noob at this, so please don't mind if I am asking stupid stuff. 
I would greatly appreciate any help.

**Comment by dallae** [comment]
> > @sekhar989 DuckDNS automatically creates a wildcard CNAME record for the subdomains. So when you create an example.duckdns.org domain, *.example.duckdns.org will automatically resolve to the same IP address
> 
> Ok ok! However, I'm still facing the issue. Please take a look at the screenshots for details. <img alt="Screenshot 2024-07-04 at 14 02 32" width="1240" src="https://private-user-images.githubusercontent.com/19548050/345757196-0be89208-3f7a-4712-ae9b-1cbe036beec6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3NTUzNDksIm5iZiI6MTcyMDc1NTA0OSwicGF0aCI6Ii8xOTU0ODA1MC8zNDU3NTcxOTYtMGJlODkyMDgtM2Y3YS00NzEyLWFlOWItMWNiZTAzNmJlZWM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEyVDAzMzA0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUxYWM5ZTY3ZmUwNDk3NzQxMzQ3YmQ2NTQwYTAwMDRhZDJkZmE5N2NhZGFhOGVkYmNkMWFhZDg2ODkzNjU2MmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HMH-yCQ8wuglOgsxspzQEqBvcx5OMz50Gv-a0uckIpU"> ![image](https://private-user-images.githubusercontent.com/19548050/345758165-2e0f2556-4ce9-45fd-9ee5-4907579008f7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3NTUzNDksIm5iZiI6MTcyMDc1NTA0OSwicGF0aCI6Ii8xOTU0ODA1MC8zNDU3NTgxNjUtMmUwZjI1NTYtNGNlOS00NWZkLTllZTUtNDkwNzU3OTAwOGY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEyVDAzMzA0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0YzRlZWRmZTAzNzUzYjIyMDUzZjQ1NTU0OWVmOGEyZTQ3ZDkxYzU0YWFiYjdmMzY5YmM4YWNiYWJjYTI3NTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xvuEMyN_XfWKGUhN-82qh0GIYU7NRzzH1kl_JcpKZOg)
> 
> Firstly, I appreciate the hard work and sharing it with us. Thank You so much for this. I'm a noob at this, so please don't mind if I am asking stupid stuff. I would greatly appreciate any help.

make sure anything else is not currently using port 80 otherwise it will fail
run the command below to see if its currently busy
sudo ss -tlnp | grep 80

**Comment by sekhar989** [comment]
Yep, I did check that. There was nothing else running on port 80. I just had to wait for some time.
It's all configured now.
However, I see that iPlayer detects my location. I checked for DNS leaks, there was nothing.
Any ideas would be appreciated.
